src/me/ryanhamshire/PopulationDensity/PopulationDensity.java

```diff
@@ -814,7 +817,7 @@ else if(cmd.getName().equalsIgnoreCase("invite") && player != null)
 			
 			//send a notification to the invitee, if he's available
 			@SuppressWarnings("deprecation")
-            Player invitee = this.getServer().getPlayer(args[0]);			
+            Player invitee = findPlayerByName(args[0]);
```

Comment:
This isn't deprecated anymore. Doesn't matter anyways - the reason it was "deprecated" in the first place was to discourage plugin makers from storing data with player names and to use UUIDs instead.

Please revert this change entirely. You may change this to `getPlayerExact` if you wish